src/app/clusters/identify-server/tests/TestIdentifyCluster.cpp

```diff
@@ -135,28 +134,28 @@ TEST_F(TestIdentifyCluster, AcceptedCommandsWithOptionalTriggerEffectTest)
 
 TEST_F(TestIdentifyCluster, ReadAttributesTest)
 {
-    chip::Test::TestServerClusterContext context;
     IdentifyCluster cluster(
         IdentifyCluster::Config(kTestEndpointId, mMockTimerDelegate).WithIdentifyType(IdentifyTypeEnum::kVisibleIndicator));
-    EXPECT_EQ(cluster.Startup(context.Get()), CHIP_NO_ERROR);
+    chip::Test::ClusterTester tester(cluster);
+    EXPECT_EQ(cluster.Startup(tester.GetServerClusterContext()), CHIP_NO_ERROR);
 
     // Read and verify IdentifyType
     uint8_t identifyType;
```

Comment:
tidy is silly here ... we need things like this throughout:


```suggestion
    uint8_t identifyType{};
```

or


```suggestion
    uint8_t identifyType = 0;
```